packages/core/src/status-adornment/StatusAdornment.css

```diff
@@ -1,4 +1,8 @@
 .saltStatusAdornment {
+  height: var(--salt-text-lineHeight);
```

Comment:
Is this to fix the icon haven't different alignment to text? If so we've already used different techniques that don't affect the component directly e.g. Banner. I spoke to the designers to other day about this design quirk.